config/default.schema.json

```diff
@@ -1586,6 +1586,15 @@
 				"CLIENT_ID": "",
 				"CLIENT_SECRET": ""
 			}
+		},
+		"HOSTNAME": {
+			"type": "string",
+			"description": "Hostname. Should not be usually defined as it's expected to be taken from the process env."
+		},
+		"HEALTHCHECKS_EXCLUDE_MONGODB": {
+			"type": "boolean",
+			"default": false,
+			"description": "Toggle that, when enabled, excludes MongoDB from the healthchecks."
```

Comment:
```suggestion
			"description": "Toggle that, when enabled, excludes MongoDB from the health checks."
```

Reply:
Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/9f7cb45820e06668af950204e05f82983c112e54 .